go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/charmbracelet/bubbletea v0.13.1
 	github.com/cpuguy83/go-md2man v1.0.10
 	github.com/fatih/color v1.10.0
+	github.com/gerow/pager v0.0.0-20190420205801-6d4a2327822f
```

Comment:
Is there an alternative to this package? It seems the last modification was done in April 2019, is it maintained?

Reply:
The only alternative I've found was updated even longer ago: github.com/sean-/pager
Well, there is a fork of that with a couple of later commits that might be an alternative: github.com/peloy/pager
But the drawback is that for that library a separate check is needed to know if a terminal is attached.

No, I don't get the impression it is maintained.

Reply:
I will take a look at the source code of the pager package tonight (after work), and then get back to this PR. I think we can merge this tonight with `gerow/pager`, but may be long term we can add similar functionality to jx-helper (since u mentioned there are no go pager packages which are maintained) to ensure this part is maintained. wdyt?

Reply:
Sure. Also the functionality is isolated and while convenient it's not crucial. So in the worst case it can simply be removed.